docs/auth0_apis_delete.md

```diff
@@ -9,10 +9,19 @@ To delete interactively, use `auth0 apis delete` with no arguments.
 
 To delete non-interactively, supply the API id and the `--force` flag to skip confirmation.
 
+## Usage
 ```
 auth0 apis delete [flags]
 ```
 
+## Examples
+
+```
+  auth0 apis delete 
+  auth0 apis delete <id|audience>
+  auth0 apis delete <id|audience> --force
```

Comment:
Should `<id|audience>` be `<api-id|api-audience>` instead, like in other resources?